linode/manager
# Revert "test: [M3-7863] - Use `happy-dom` instead of `jsdom` in unit tests (#11085)"

## Description 📝
- Reverts #11085 (commit efa85802c21e63e41dbce3517826090c8486e6dd)
- Unfortunately, `happy-dom` **did** end up introducing significant unit test fake for us particularly in our coverage script
- View @abailly-akamai 's internal Slack thread for more details on the findings/impact 🔍 

## How to test 🧪

- Verify GitHub Actions pass (both the unit test job and the overage jobs)
  - I believe the `base_branch` coverage job failure is okay because that is what this PR will fix once it is in develop
- Run some tests locally to check for stability (`yarn test`)
- Run the coverage script locally and verify (`yarn coverage`)

## As an Author I have considered 🤔
